README.md

```diff
@@ -159,6 +159,8 @@ Rate limiting uses [lambda-rate-limiter](https://github.com/blackflux/lambda-rat
 
 To customize rate limiting, the package options are passed as `limiter` into the constructor.
 
+By default rate limiting uses the client ip (`['requestContext.identity.sourceIp']`). This can be customized by using `rateLimitTokenPaths` as an array (evaluated left to right) to target different keys on the lambda event. If rate limit is enabled and no valid rate limit token is detected, an exception is thrown. 
```

Comment:
```suggestion
By default rate limiting uses the client ip (`['requestContext.identity.sourceIp']`). This can be customized by using `rateLimitTokenPaths` as an array (evaluated left to right) to target different keys on the lambda event. If rate limit is enabled and no valid rate limit token is detected on the event, an exception is thrown. 
```